src/server/search/search_family_test.cc

```diff
@@ -428,4 +428,47 @@ TEST_F(SearchFamilyTest, FtProfile) {
   }
 }
 
+TEST_F(SearchFamilyTest, BasicSort) {
+  auto AreRange = [](size_t total, size_t l, size_t r, string_view prefix) {
+    vector<string> out;
+    for (size_t i = min(l, r); i < max(l, r); i++)
+      out.push_back(absl::StrCat(prefix, i));
+    if (l > r)
+      reverse(out.begin(), out.end());
+    return DocIds(total, out);
+  };
+
+  max_memory_limit = INT_MAX;
+
```

Comment:
This freaked me out, I get `out of memory`. The test already starts on 150mb fresh 😵 Will look into out